handlers/getItems.go

```diff
@@ -0,0 +1,61 @@
+package handlers
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/mount-joy/thelist-lambda/data"
+	"github.com/mount-joy/thelist-lambda/db"
+)
+
+type getItems struct {
+	db db.DB
+}
+
+func newGetItems() RouteHandler {
+	return &getItems{
+		db: db.NewDB(),
```

Comment:
Given that DB will I assume eventually do more than GetItems, the naming of some of this might need thinking about, otherwise we are going to end up with a `db.NewDB()` on a `deleteItems` a `putItems` etc.

Reply:
I think this is sorted 👍